test/api_test.mjs

```diff
@@ -389,3 +389,22 @@ test("handling failed job", async (t) => {
   t.truthy(res.error);
   t.true(res.error.includes("MockError"));
 });
+
+test("sending a arweave message", async (t) => {
+  const message = {
+    type: "arweave",
+    version: messages.version,
+    options: {
+      uri: "ar://ltmVC0dpe7_KxFHj0-S7mdvXSfmcJOec4_OfjwSzLRk/1",
+      gateway: "https://arweave.net",
+    },
+    results: null,
+    error: null,
+  };
+
+  const res = await messages.route(message);
```

Comment:
I'm a bit confused how this test can pass considering that the arweave message schema is only adopted in neume-network/schema@0.8.0, so wouldn't this message be considered as invalid?